lib/processExportsField.js

```diff
@@ -8,7 +8,7 @@
 /** @typedef {string|string[]} DirectMapping */
 /** @typedef {{[k: string]: MappingValue}} ConditionalMapping */
 /** @typedef {ConditionalMapping|DirectMapping|null} MappingValue */
-/** @typedef {Record<string, MappingValue>|string} ExportField */
+/** @typedef {Record<string, MappingValue>|ConditionalMapping|string} ExportField */
```

Comment:
```suggestion
/** @typedef {Record<string, MappingValue>|ConditionalMapping|DirectMapping} ExportField */
```

I guess `"exports": ["./a", "./b"]` is also something valid that need to be handled.